examples/textual_inversion/textual_inversion.py

```diff
@@ -270,11 +270,11 @@ def parse_args():
         help="Number of images that should be generated during validation with `validation_prompt`.",
     )
     parser.add_argument(
-        "--validation_epochs",
+        "--validation_steps",
```

Comment:
Using `steps` here (and in dreambooth) script makes sense to me and also it aligns well with `save_steps`. But Removing the existing `arg` would be a breaking change, 

 Instead of changing the arg, maybe we could introduce an additional arg called `validation_steps` and allow only one of them to be set. Inside the script, we always use `validation_steps` , we can compute that easily using the epochs value. And to align saving and validating we could prefer the `validation_steps` argument in the docs.

wdyt @patrickvonplaten @williamberman @pcuenca ?

Reply:
@patil-suraj good point! One point that I have is to do this we might want to make the logging a function(because calling in 2 places)

But if we do that we need to pass wandb as an argument as we aren't doing the import in the global scope. I was thinking of doing this before but it felt a bit hacky. Happy to show what I mean!

Reply:
Preferring config via steps over epochs makes sense to me 👍

Reply:
Steps also makes sense to me but let's deprecate `validation_epochs` nicely. Could we keep `validation_epochs` and just add a new `validation_steps` . Then will convert validation_epochs to validation_steps if used and throw a deprecation warning. Would that be ok for you @isamu-isozaki ?

Reply:
@patrickvonplaten yup sounds good! Then will move the code to a function so it's cleaner